src/components/Post.js

```diff
@@ -44,7 +60,11 @@ class Post extends Component {
                     })
                   : null}
               </ul>
-              <Comments comments={this.props.post.comments} postUuid={this.props.post.entry.uuid} />
+              <Comments
+                comments={this.props.post.comments}
+                postUuid={this.props.post.entry.uuid}
+                user={this.props.profile}
```

Comment:
Be consistent across the app. Choose one: `user` or `profile`. Or even better, if the role is `currentUser`, then use that.